app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitViewFragment.java

```diff
@@ -65,6 +70,37 @@ public void onClick(View view) {
             }
         });
 
+
+        //all of the non-clickable buttons
+        Button mondayButton = view.findViewById(R.id.monday_button);
+        if (tracker.getMonday()){
+            mondayButton.performClick();
+        }
+        Button tuesdayButton = view.findViewById(R.id.tuesday_button);
+        if (tracker.getTuesday()){
+            tuesdayButton.performClick();
+        }
+        Button wednesdayButton = view.findViewById(R.id.wednesday_button);
+        if (tracker.getWednesday()){
+            wednesdayButton.performClick();
+        }
+        Button thursdayButton = view.findViewById(R.id.thursday_button);
+        if (tracker.getThursday()){
+            thursdayButton.performClick();
+        }
+        Button fridayButton = view.findViewById(R.id.friday_button);
+        if (tracker.getFriday()){
+            fridayButton.performClick();
+        }
+        Button saturdayButton = view.findViewById(R.id.saturday_button);
+        if (tracker.getSaturday()){
+            saturdayButton.performClick();
+        }
+        Button sundayButton = view.findViewById(R.id.sunday_button);
+        if (tracker.getSunday()){
+            sundayButton.performClick();
+        }
+
```

Comment:
This sets all the buttons according to the string from the DB.

Reply:
Extract to method